packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -252,6 +255,29 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             ...this.connectionManager.connectionProps,
         };
     }
+  /**
+     * Returns set of props that can be logged in telemetry that provide some insights / statistics
+     * about current or last connection (if there is no connection at the moment)
+    */
+    public connectionProps(): ITelemetryProperties {
```

Comment:
I believe you want to modify existing connectionProps getter above. Adding this function will do nothing as nobody will call it.